LibreNMS/Alert/Transport/Matrix.php

```diff
@@ -54,7 +54,7 @@ private function contactMatrix($obj, $server, $room, $authtoken, $message)
 
         $message = SimpleTemplate::parse($message, $obj);
 
-        $body = ['body'=>$message, 'msgtype'=>'m.text'];
+        $body = ['body'=>strip_tags($message), 'formatted_body'=>"$message", 'msgtype'=>'m.text', 'format'=>'org.matrix.custom.html'];
```

Comment:
This looks odd but is just like the example on Matrix API docs. Did you test how this behaves with and without the formatting?

Reply:
From https://spec.matrix.org/v1.3/client-server-api/

> Some message types support HTML in the event content that clients should prefer to display if available. Currently m.text, m.emote, and m.notice support an additional format parameter of org.matrix.custom.html. When this field is present, a formatted_body with the HTML must be provided. The plain text version of the HTML should be provided in the body.

> Clients should limit the HTML they render to avoid Cross-Site Scripting, HTML injection, and similar attacks. The strongly suggested set of HTML tags to permit, denying the use and rendering of anything else, is: font, del, h1, h2, h3, h4, h5, h6, blockquote, p, a, ul, ol, sup, sub, li, b, i, u, strong, em, strike, code, hr, br, div, table, thead, tbody, tr, th, td, caption, pre, span, img, details, summary.